rebasehelper/tests/test_specfile.py

```diff
@@ -460,3 +461,13 @@ def test_pypi_to_python_hosted_url_trans(self):
         assert 'https://pypi.python.org/' in self.SPEC_FILE_OBJECT._get_raw_source_string(7)
         spec_hooks_runner.run_spec_hooks(None, self.SPEC_FILE_OBJECT)
         assert 'https://files.pythonhosted.org/' in self.SPEC_FILE_OBJECT._get_raw_source_string(7)
+
+    def test_update_paths_to_patches(self):
+        """
+        Check updated paths to patches in the rebased directory
+        :return:
+        """
+        self.SPEC_FILE_OBJECT.update_paths_to_patches()
+
+        for line in self.SPEC_FILE_OBJECT.spec_content:
+            assert not re.search('^Patch.*rebased_sources/', line)
```

Comment:
Right, I didn't notice that. But I still think that you should first find the `Patch5` line and then make assertion on it, rather than doing that for every line.